fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -28,6 +28,13 @@ server {
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Could we add a comment on this block to indicate it only stays here for retro-compatibility ?

Reply:
Being curious : retro-compatibility with what ?

Reply:
Retro-compatibility for people who may be using `https://fr.openfisca.org/api/v2`

Reply:
If we don't want retro-compatibility then this block should be removed (once the leg explorer is merged).